files/default/slurm/templates/slurm_parallelcluster_queue_partition.conf

```diff
@@ -0,0 +1,30 @@
+# This file is automatically generated by pcluster
+
+{% for instance in queue.instances %}
+{%- set has_static_nodes = true if instance.static_size > 0 else false -%}
+{%- set has_dynamic_nodes = true if instance.dynamic_size > 0 else false -%}
+{% if has_static_nodes -%}
+NodeName={{ queue.queue_name }}-static-{{ instance.type }}[1-{{ instance.static_size }}] CPUs={{ instance.vcpus }} State=CLOUD Feature=static,{{ instance.type }} {% if instance.gpus > 0 %}Gres=gpu:tesla:{{ instance.gpus }}
+{%- endif %}
+{% endif -%}
+{% if has_dynamic_nodes -%}
+NodeName={{ queue.queue_name }}-dynamic-{{ instance.type }}[1-{{ instance.dynamic_size }}] CPUs={{ instance.vcpus }} State=CLOUD Feature=dynamic,{{ instance.type }} {% if instance.gpus > 0 %}Gres=gpu:tesla:{{ instance.gpus }}
+{%- endif %}
+{% endif -%}
+{% endfor %}
+
+NodeSet={{ queue.queue_name }}_nodes Nodes={%- for instance in queue.instances -%}
+{%- set has_static_nodes = true if instance.static_size > 0 else false -%}
+{%- set has_dynamic_nodes = true if instance.dynamic_size > 0 else false -%}
+{%- if has_static_nodes and has_dynamic_nodes -%}
+{{ queue.queue_name }}-static-{{ instance.type }}[1-{{ instance.static_size }}],{{ queue.queue_name }}-dynamic-{{ instance.type }}[1-{{ instance.dynamic_size }}]
+{%- elif has_static_nodes -%}
+{{ queue.queue_name }}-static-{{ instance.type }}[1-{{ instance.static_size }}]
+{%- elif has_dynamic_nodes -%}
+{{ queue.queue_name }}-dynamic-{{ instance.type }}[1-{{ instance.dynamic_size }}]
+{%- endif -%}
+{{- "," if not loop.last else "" -}}
+{%- endfor %}
+
+PartitionName={{ queue.queue_name }} Nodes={{ queue.queue_name }}_nodes MaxTime=INFINITE State=UP
```

Comment:
we probably want to set the first queue/partition as default

Reply:
Setting default in `slurm_parallelcluster.conf`